cryptographic_estimators/SDEstimator/SDAlgorithms/bjmm.py

```diff
@@ -30,29 +38,25 @@
 
 class BJMM(SDAlgorithm):
     def __init__(self, problem: SDProblem, **kwargs):
-        """
-        Complexity estimate of BJMM algorithm in depth 2,3
+        """Complexity estimate of BJMM algorithm in depth 2,3.
 
-        The algorithm was introduced in [BJMM12]_  as an extension of [MMT11]_.
+        The algorithm was introduced in [BJMM12] as an extension of [MMT11].
 
-        expected weight distribution::
+        The expected weight distribution is:
 
-            +--------------------------+-------------------+-------------------+
-            | <-----+ n - k - l +----->|<--+ (k + l)/2 +-->|<--+ (k + l)/2 +-->|
-            |           w - 2p         |        p          |        p          |
-            +--------------------------+-------------------+-------------------+
+        +---------------------------+-------------------+-------------------+
```

Comment:
the indentation should be kept (in line with the above at least) and for sphincs this ensured being displayed in a codeblock without spaces being removed